projects/pastanaga-angular/src/lib/tabs/tabs-list.component.ts

```diff
@@ -27,11 +27,20 @@ export class TabsListComponent implements AfterContentInit, OnDestroy {
         this.updateSlider();
     }
 
+    @Input() set defaultOnMobile(value: boolean) {
+        this._defaultOnMobile = coerceBooleanProperty(value);
+    }
+
+    get defaultOnMobile() {
+        return this._defaultOnMobile;
+    }
+
```

Comment:
Reading it, I feel like you inverted the logic. By default on mobile, for now we keep the tabs as a list (so we don't need to change the ther implementations). So we need an option to keep the tabs as tabs on mobile. Something like `displayasTabOnMobile` which is false by default, and will be true where you need it.

Reply:
Indeed, update both variable and docs.